bin/HFO

```diff
@@ -118,6 +120,10 @@ def parseArgs():
                  help='Number of offensive uncontrolled players. Default: 0.')
   p.add_argument('--defense-npcs', dest='defenseNPCs', type=int, default=0,
                  help='Number of defensive uncontrolled players. Default: 0.')
+  p.add_argument('--offense-team', dest='offenseTeam', type=str, default='left',
+                 help='Team name to pull offensive NPCs from. Default: left.') 
+  p.add_argument('--defense-team', dest='defenseTeam', type=str, default='right',
+                 help='Team name to pull defensive NPCs from. Default: right')
```

Comment:
Oh, I changed the way it searches for team folders since that prevents having 2 teams from the same binary to play each other. The mapping is in Trainer.py under createTeam()

Similar thing for the available teams for help message -- currently, anything that contains "HELIOS" will try to launch the helios binary, and anything else will launch agent2d. I'll update the message